kernel/acpi/src/lib.rs

```diff
@@ -131,7 +117,7 @@ pub fn init(rsdp_address: Option<PhysicalAddress>, page_table: &mut PageTable) -
                             debug!("                  path: {:?}", dev_scope.path());
                         }
                     }
-                    _ => { }
+                    _ => { error!("other DMAR entries are currently unhandled") }
```

Comment:
^

Reply:
Is this logging statement necessary? I don't think it's an error to encounter an unknown DMAR entry.

Reply:
agreed, we don't use them yet so let's keep it as is.

Reply:
warn would have made more sense, but I guess adding an `allow` here does the same thing until we need the other entries

Reply:
you could lower it to a `debug` or `trace` if you like, but we really don't need to be logging this since it doesn't provide any useful info atm.